examples/sample.go

```diff
@@ -39,6 +53,19 @@ func initData() {
 		fmt.Println("Error Code:", status)
 		os.Exit(1)
 	}
+	// set the actual channel of the release e.g stable
+	status = lexactivator.SetReleaseChannel("RELEASE_CHANNEL")
```

Comment:
remove the release channel and platform functions

Reply:
release verison is missing

Reply:
Done, the release version is already there.